### PR TITLE
Arch support

### PR DESCRIPTION
**Pull request by teiuaz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,6 @@
 /imgui-master/
 /googletest/
 /spdlog/
+SPRV
+cmake_build
+logs
```
Comment (N1kla3) [inline .gitignore:11]: Logs folder will be generated by spdlog at application location, so application will be in cmake build folder. Which is already gitignore

File: CMakeLists.txt [hunk 2]
```diff
@@ -56,10 +56,10 @@ list(TRANSFORM SOURCE PREPEND "src/")
 enable_testing()
 
 add_executable(renderer src/main.cpp ${SOURCE} ${IMGUI})
-target_compile_definitions(renderer PRIVATE VK_USE_PLATFORM_WIN32_KHR)
+target_compile_definitions(renderer PRIVATE)
```
Comment (N1kla3) [inline CMakeLists.txt:59]: if we won't WIN32 better to delete this line. It make no sense without any arguments

File: README.md [hunk 3]
```diff
@@ -7,4 +7,26 @@
...
+On linux
+```bash
+./compile_shaders.sh
+```
+
+## Build
+```bash
+mkdir cmake_build
+cmake -S . -B cmake_build
+cmake --build cmake_build
```
Comment (N1kla3) [inline README.md:25]: In clion I can specify release or debug build in IDE, and cmake will generate two folders. But here you generate one build type, and without specs. I think we need add more configs to cmake then.

File: compile_shaders.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+dir=SPRV
+mkdir -p $dir
+glslc shaders/shader.frag -o $dir/frag.spv
```
Comment (N1kla3) [inline compile_shaders.sh:3]: very good

File: README.md (hunk 3 above)
Comment (N1kla3) [reply README.md:25]: https://stackoverflow.com/questions/27086145/what-is-the-default-build-configuration-of-cmake#:~:text=In%20this%20answer%2C%20it%20says,the%20default%20cmake%20build%20configuration.

**Review by N1kla3 — APPROVED** [review]
This looks fine, just some recomendations. There is no anything crucial to change

File: .gitignore (hunk 1 above)
Comment (teiuaz) [reply .gitignore:11]: у мяне гэта папка генерыцца у корне

**Pull request merged**